src/main/resources/application.conf

```diff
@@ -30,6 +32,16 @@ nav {
             expiration_in_days = "",
             expiration_in_days = ${?S3_EXPIRATION_IN_DAYS}
         }
+        gcp_bucket {
+            service_endpoint = ""
+            service_endpoint = ${?S3_SERVICE_ENDPOINT}
```

Comment:
S3 ?

Reply:
Det trengs egentlig ikke. Har glemt å fjerne den. Antok at man trengte noe endpoint for å koble til riktig bucket, men trenger visst ingenting for å å få det til. Da endpoint er innebygget i GSC (Google cloud storage) klienten. Trenger bare å spesifisere bucket.